### PR TITLE
Docs: add reference to flog CLI args

### PR DESCRIPTION
**Pull request by dapeleg-dn**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [x] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)

### CODE REVIEW

File: docs/sources/getting-started/_index.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Choose one of these two ways to apply a new configuration:
         ```
         docker-compose down
         ```
-    1. Edit the `docker-compose.yaml` file.  Within the YAML file, change the `flog.command` field's value to specify your flog output.
+    1. Edit the `docker-compose.yaml` file.  Within the YAML file, change the `flog.command` field's value to specify your flog output. See the `flog` command line arguments here: https://hub.docker.com/r/mingrammer/flog
```
Comment (JStickler) [inline docs/sources/getting-started/_index.md:125]:
```suggestion
    1. Edit the `docker-compose.yaml` file.  Within the YAML file, change the `flog.command` field's value to specify your flog output. Refer to the `flog` (command line arguments)[https://hub.docker.com/r/mingrammer/flog].
```

**Review by JStickler — COMMENTED** [review]
[Docs team]  Our standard is to provide some sort of [link text ](https://grafana.com/docs/writers-toolkit/writing-guide/useful-link-text/) for URLs.

**Comment by dapeleg-dn** [comment]
> [Docs team] Our standard is to provide some sort of [link text ](https://grafana.com/docs/writers-toolkit/writing-guide/useful-link-text/) for URLs.

Updated.

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

**Comment by MasslessParticle** [comment]
@dapeleg-dn We'd be happy to pull this in. Can you fix the merge conflicts?

**Comment by dapeleg-dn** [comment]
> @dapeleg-dn We'd be happy to pull this in. Can you fix the merge conflicts?

I tried to rebase and push. I didn't see any conflict, though; I have just added some minimal text to the docs. 

Now I see a failure during the build. Can you help me understand what's wrong? 
https://drone.grafana.net/grafana/loki/24903/3/13

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

**Comment by JStickler** [comment]
Sorry, I should read the comments when I come back from a long weekend and approve a PR I've already approved.  😛

>Now I see a failure during the build. Can you help me understand what's wrong?

It looks like changes made in another PR probably need to be pulled in to pass the validation checks.  You need to 1) rebase and then 2) Update the generated documentation by running 'make doc' and committing the changes.  

Let's see if that gets us past the checks.

**Comment by dapeleg-dn** [comment]
Never mind. I just wanted to contribute a little single sentence to the documentation. If the procedure is so complicated, I will leave that.

**Comment by JStickler** [comment]
@dapeleg-dn  Thanks for being patient with all the CI checks!  Really appreciate it.

**Pull request merged**